voyager-publish/bin/publish-batch.js

```diff
@@ -68,12 +68,27 @@ const ieContract = new ethers.Contract(
   provider
 ).connect(signer)
 
+const addPidToConsoleArgs = (...args) => {
+  if (typeof args[0] === 'string') {
+    args[0] = `[worker:%s] ${args[0]}`
+    args.splice(1, 0, process.pid)
+  } else {
+    args.unshift(`[worker:${process.pid}]`)
+  }
```

Comment:
```suggestion
  args.unshift(`[worker:${process.pid}]`)
```

I think that should create the same result

Reply:
no scratch that it doesn't, because of fprintf formatting. But I think I have a different idea

Reply:
see next review